### PR TITLE
Update substrate libraries

### PR DESCRIPTION
**Pull request by yabirgb**
This update should improve the speed for arm build since new wheels are distributed for arm

### CODE REVIEW

**Comment by yabirgb** [comment]
Failing tests are due to maintenance in gemini

File: Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ COPY ./requirements.txt /app/requirements.txt
 
 WORKDIR /app
 
-RUN pip install maturin==0.12.3 py-sr25519-bindings==0.1.3 py-ed25519-bindings==0.1.3 py-bip39-bindings==0.1.7 wheel
+RUN pip install maturin==0.12.3 py-sr25519-bindings==0.1.4 py-ed25519-bindings==1.0.1 py-bip39-bindings==0.1.8 wheel
```
Comment (LefterisJP) [inline Dockerfile:39]: Hmmm question here. This is dangerous. Why not make it take it from requirements.txt? I am quite sure this and requirements.tx will diverge at some point by mistake

Comment (yabirgb) [reply Dockerfile:39]: Didn't know it existed but maybe we can use `sys_platform == "arm64"`

**Review by LefterisJP — COMMENTED** [review]
Did you check this works with the build and packages properly @yabirgb ?

**Comment by yabirgb** [comment]
> Did you check this works with the build and packages properly @yabirgb ?

I checked the docker builds and work

**Review by LefterisJP — APPROVED** [review]
Okay so let's take a chance here and see if it will work. I am a bit reluctant as I did not try it myself.

**Pull request merged**